CMakeLists.txt

```diff
@@ -178,7 +178,7 @@ if(${TRITON_ENABLE_GPU})
   target_link_libraries(
     triton-backend-utils
     PUBLIC
-      CUDA::cudart
+      CUDA::cudart_static
```

Comment:
Since the library needs to be link at the finally build step, the backend still need to link to cudart explicitly, regardless of static or not. From the backend perspective, wouldn't it make more sense if the CUDA libraries are linked in the same manner (all shared / all static), perhaps we can provide cmake option so users can specify whether they want the utilites to link against the static or the shared version of CUDA llibraries.